_posts/2015-09-08-code-talks.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "ePages talks REST API"
+date: "2015-09-08 08:04:00"
+icon: code
+tags: misc
+categories: tech-stories conferences api
+authors: ["Birgit"]
+---
+
+Have you signed up for `code.talks` ?
+Would you like to learn more about the challenges of documenting REST APIs?
+We will be happy to see you on our talk then.
+Interested in some key points?
+Here you go:
+
+* RAML vs Spring REST Docs
+* Generating the docs
```

Comment:
sounds so generic, I'd rather leave it off - or making it more specific (documentation workflow?)